test/es6/rlexe.xml

```diff
@@ -100,6 +100,12 @@
       <compile-flags>-args summary -endargs</compile-flags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>unscopablesWithEsArrayFindFromLastTest.js</files>
+      <compile-flags>-ESArrayFindFromLast -args summary -endargs</compile-flags>
+    </default>
+  </test>
```

Comment:
Instead of adding this as a new test could update the existing unscopables test and add the flag to it.

Reply:
I think we'd better to keep original behavior if it's behind the flag.

Reply:
And we can add these into original test if unflagged this.

Reply:
OK - just means we have to remember to update the original test when unflagging.